frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/resource-status.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Badge } from '@patternfly/react-core';
+import * as classNames from 'classnames';
+import '../_resource.scss';
```

Comment:
not sure if we have a convention of the file name with `_*.scss`, and if these styles are only applicable to this component then can have style file name same as component one and at same level